docs/end-to-end-demo.md

```diff
@@ -82,6 +82,10 @@ carry out one or the other, then continue on with the rest of the steps.
     curl http://deb.debian.org/debian/pool/main/v/v4l2loopback/v4l2loopback-dkms_0.12.5-1_all.deb -o v4l2loopback-dkms_0.12.5-1_all.deb
     sudo dpkg -i v4l2loopback-dkms_0.12.5-1_all.deb
     ```
+1. Install the prerequisites of v4l2loopback: it needs to be loaded simultaneously with the other kernel module 'videodev' contained in linux-modules-extra
```

Comment:
Can you add this to the previous step to reduce the total number of steps?

Reply:
Also, one more request. Testing this on fresh Ubuntu 20.04, package list needs to be updated and dkms installed. Can you please include these:
```
sudo apt update
sudo apt -y install linux-modules-extra-$(uname -r)
sudo apt -y install dkms
```